### PR TITLE
Fixed building on Windows

### PR DESCRIPTION
**Pull request by mrpimpunicorn**
Minor rendering improvement

### CODE REVIEW

File: TrueCraft.Launcher/TrueCraft.Launcher.csproj [hunk 1]
```diff
@@ -110,6 +110,9 @@
...
     <!-- TODO: Someone needs to fix this up -->
-    <PostBuildEvent>rm $(TargetDir)/MonoGame.Framework.dll &amp;&amp; cp $(SolutionDir)/packages/MonoGame.Framework.Linux.3.4.0.459/lib/net40/MonoGame.Framework.dll $(TargetDir)/MonoGame.Framework.Linux.dll &amp;&amp; cp $(SolutionDir)/packages/MonoGame.Framework.WindowsGL.3.4.0.459/lib/net40/MonoGame.Framework.dll $(TargetDir)/MonoGame.Framework.Windows.dll</PostBuildEvent>
+    <PostBuildEvent>del $(TargetDir)/MonoGame.Framework.dll &amp;&amp; copy $(SolutionDir)/packages/MonoGame.Framework.Linux.3.4.0.459/lib/net40/MonoGame.Framework.dll $(TargetDir)/MonoGame.Framework.Linux.dll &amp;&amp; cp $(SolutionDir)/packages/MonoGame.Framework.WindowsGL.3.4.0.459/lib/net40/MonoGame.Framework.dll $(TargetDir)/MonoGame.Framework.Windows.dll</PostBuildEvent>
+    <PostBuildEvent>del $(TargetDir)MonoGame.Framework.dll
+copy $(SolutionDir)packages\MonoGame.Framework.Linux.3.4.0.459\lib\net40\MonoGame.Framework.dll $(TargetDir)MonoGame.Framework.Linux.dll
+copy $(SolutionDir)packages\MonoGame.Framework.WindowsGL.3.4.0.459\lib\net40\MonoGame.Framework.dll $(TargetDir)MonoGame.Framework.Windows.dll</PostBuildEvent>
```
Comment (SirCmpwn) [inline TrueCraft.Launcher/TrueCraft.Launcher.csproj]: Twice?

Comment (mrpimpunicorn) [inline TrueCraft.Launcher/TrueCraft.Launcher.csproj]: Ouch, lemme fix.

**Comment by SirCmpwn** [comment]
TrueCraftGame.cs(342,9): error CS1922: A field or property `Microsoft.Xna.Framework.Graphics.BlendState' cannot be initialized with a collection object initializer because type `Microsoft.Xna.Framework.Graphics.BlendState' does not implement `System.Collections.IEnumerable' interface

**Comment by SirCmpwn** [comment]
Does this compile for you?

**Comment by mrpimpunicorn** [comment]
Nope, I'll push a fix. :zzz:

**Comment by SirCmpwn** [comment]
Danke

**Comment by SirCmpwn** [comment]
You should probably come hang out in IRC, by the way. #truecraft on irc.esper.net.

**Comment by mrpimpunicorn** [comment]
Sure, why not.

**Pull request merged**